content/docs/2.5/scalers/newrelic.md

```diff
@@ -35,11 +35,11 @@ triggers:
 
 - `nrAccount` - The account within Newrelic that the request should be targeted against.
 - `nrQueryKey` - The API key that will be leveraged to connect to Newrelic and make requests. [official documentation](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/)
-- `nrRegion` - The region to connect to for the Newrelic apis. (Values: `LOCAL`, `EU`, `STAGING`, `US`)
+- `nrRegion` - The region to connect to for the Newrelic apis. (Values: `LOCAL`, `EU`, `STAGING`, `US`, Default: `US`)
 - `metricName` - The metric to pull from the query result.
 - `nrql` - The Newrelic query that will be run to get the data requested. [official documentation](https://docs.newrelic.com/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/)
 - `threshold` - A threshold that is used as the `targetAverageValue` in the HPA configuration.
-- `nrLogLevel` - The logging level of the underlying Newrelic class to aid in debugging. (Values: `debug`, `error`, `fatal`, `info`, `trace`, `warn`, Optional)
+- `nrLogLevel` - The logging level of the underlying Newrelic class to aid in debugging. (Values: `debug`, `error`, `fatal`, `info`, `trace`, `warn`, Optional, Default: `info`)
```

Comment:
Please, according our convention, move default between Values and Optional:
[`- name - Description. (Values: x, y, z, Default: y, Optional, Extra Info)`](https://github.com/kedacore/keda-docs#writing-documentation-for-a-scaler)